Library/Homebrew/style.rb

```diff
@@ -242,6 +261,12 @@ def self.run_shfmt(files, fix: false)
       $CHILD_STATUS.success?
     end
 
+    def self.run_actionlint(files)
+      files = github_workflow_files if files.blank?
+      system actionlint, "-shellcheck", shellcheck, "-config-file", HOMEBREW_REPOSITORY/".github/actionlint.yaml", *files
```

Comment:
```suggestion
      system actionlint, "-shellcheck", shellcheck,
                         "-config-file", HOMEBREW_REPOSITORY/".github/actionlint.yaml",
                         *files
```
Oops, line is too long